llvm/test/CodeGen/AArch64/reduce-or-opt.ll

```diff
@@ -0,0 +1,195 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -mtriple=aarch64-none-linux-gnu -mattr=+sve | FileCheck %s
```

Comment:
The transform is done on IR in CodeGenPrepare, right? It would probably be easier to see what is going on exactly if there would be at least some IR tests in `llvm/test/Transforms/CodeGenPrepare/AArch64`?

Reply:
Ah that's a good suggestion. To be honest I didn't even know that directory existed until now!